verisignmdns/README.md

```diff
@@ -0,0 +1,9 @@
+Verisign MDNS Provider enables access to Verisign MDNS through Denominator. This provider lets you perform zone list and resource record set management either using Denominator cli or through java API.
+
+This provider currently supports 'put', 'iterator', 'iterate by name and type', 'get by name and type', and  'delete by name and type' operations for CNAME, A, AAAA, NS, NAPTR, PTR, MX, DS, SRV and TXT record types. 
```

Comment:
Since this file is more likely to mislead others or cause maintenance, I would remove it.

For example, the supported record types is available via the api. Bugs such as not yet supporting pagination, will hopefully be fixed before release. Also, the fact that you need an account to use the provider is implied, unless there's something special about the phrase "direct user".

The best file, if there is one, would include a url to sign up for the service, and any instructions that need care.  ex. https://github.com/Netflix/denominator/tree/master/discoverydns